third_party/terraform/website/docs/d/datasource_google_service_account.html.markdown

```diff
@@ -26,15 +26,15 @@ data "google_service_account" "myaccount" {
 }
 
 resource "google_service_account_key" "mykey" {
-  service_account_id = "${data.google_service_account.myaccount.name}"
+  service_account_id = data.google_service_account.myaccount.name
 }
 
 resource "kubernetes_secret" "google-application-credentials" {
-  metadata = {
+  metadata {
     name = "google-application-credentials"
   }
-  data {
-    credentials.json = "${base64decode(google_service_account_key.mykey.private_key)}"
+  data = {
```

Comment:
the `=` is because it's a map?

Reply:
Yes, I believe so.  I mostly just ran `terraform 0.12upgrade` over all of these blocks, and this is what it spit out, but that seems to be the case here :)